documentation/Installing-Clear-Containers-on-Centos-7.md

```diff
@@ -0,0 +1,171 @@
+	- http://download.opensuse.org/repositories/home:/clearlinux:/preview:/clear-containers-2.1/CentOS_7/x86_64/
+- Install the packages:
+
+	 ```
+	 $ sudo rpm -ivh linux-container-*.x86_64.rpm clear-containers-image-*.x86_64.rpm
+	 ```
+
+**WARNING:** You should repeat this step regularly to ensure your Clear Containers are running with the latest versions of these packages.
+
+
+## Install latest Golang
+
+cc-oci-runtime supports golang 1.7 and 1.8
+The next instructions were taken from [golang site](https://golang.org/doc/install)
+
+```
+$ curl -L -O https://storage.googleapis.com/golang/go1.8.linux-amd64.tar.gz
+$ sudo tar -C /usr/local -xzf go1.8.linux-amd64.tar.gz
+$ export PATH=$PATH:/usr/local/go/bin
+$ mkdir ~/go
+$ export GOPATH=~/go
+
+```
+
+## Install the clear containers runtime from sources
+
+```
+$ export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:$PKG_CONFIG_PATH
+$ export PATH=/usr/local/gcc-6.2.0/bin:$PATH
+$ sudo yum install libuuid-devel pcre-devel libffi-devel zlib-devel gettext-devel pkgconfig libmnl-devel
+$ go get github.com/01org/cc-oci-runtime
+$ pushd go/src/github.com/01org/cc-oci-runtime
+$ git checkout tags/2.1.0
+$ ./autogen.sh --prefix=/usr/local --disable-tests --disable-functional-tests --with-cc-image=/usr/share/clear-containers/clear-containers.img --with-cc-kernel=/usr/share/clear-containers/vmlinux.container --with-qemu-path=/usr/local/bin/qemu-system-x86_64
+$ make -j5 && sudo make install
+$ popd
+
+```
+
+## Install docker 1.12
+
+```
+$ sudo tee /etc/yum.repos.d/docker.repo <<-'EOF'
+[dockerrepo]
+name=Docker Repository
+baseurl=https://yum.dockerproject.org/repo/main/centos/7/
+enabled=1
+gpgcheck=1
+gpgkey=https://yum.dockerproject.org/gpg
+EOF
+
+$ sudo yum install docker-engine-1.12.1-1.el7.centos.x86_64 docker-engine-selinux-1.12.1-1.el7.centos.noarch
+
+```
+
+For more information on docker installation, see:
+- https://docs.docker.com/engine/installation/linux/centos/
+
+
+## Configure docker to use Clear Containers by default
+
+```
+$ sudo mkdir -p /etc/systemd/system/docker.service.d/
+$ sudo vi /etc/systemd/system/docker.service.d/clr-containers.conf
+[Service]
+ExecStart=
+ExecStart=/usr/bin/dockerd -D --add-runtime cor=/usr/local/bin/cc-oci-runtime --default-runtime=cor
+```
+
+## Restart the docker systemd service
+
+```
+$ sudo systemctl daemon-reload
+$ sudo systemctl restart docker
+```
+
+## Start cc-proxy socket
+
+```
+$ sudo systemctl start cc-proxy.socket
+```
+
+## Run a Clear Container
+You are now ready to run clear containers. For example:
+
+```
+$ sudo docker run -ti fedora bash
+```
+
+## Known Issues
+
+Currently `cc-oci-runtime v2.1` only works correctly with `docker-engine=1.12.1-1.el7.centos.x86_64`. While we update `cc-oci-runtime` to work with later versions of docker, we recommend to mark `docker-engine` to not being updated on the next upgrade.
```

Comment:
"While we update `cc-oci-runtime` to work with later versions of docker, we recommend running the command below to stop the `docker-engine` package from being updated when new package versions become available. Note that there is a risk in doing this since newer versions of Docker may contain security fixes."